alacritty_terminal/src/clipboard.rs

```diff
@@ -19,10 +19,14 @@ use log::{debug, warn};
 
 use copypasta::nop_clipboard::NopClipboardContext;
 #[cfg(not(any(target_os = "macos", target_os = "windows")))]
+#[cfg(feature = "wayland")]
 use copypasta::wayland_clipboard;
 #[cfg(not(any(target_os = "macos", target_os = "windows")))]
+#[cfg(feature = "x11")]
```

Comment:
Can we combine these two statements?

Reply:
Not really, you can't have multiple cfg's.